packages/api/src/repository-infos.ts

```diff
@@ -0,0 +1,32 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { fromUrl } from 'hosted-git-info';
+
+import rootPackage from '../../../package.json' with { type: 'json' };
+
+export const REPOSITORY_URL = rootPackage.repository;
+
+const infos = fromUrl(REPOSITORY_URL);
+
+if (!infos) {
+  throw new Error(`Could not parse repository information from URL: ${REPOSITORY_URL}`);
+}
+
```

Comment:
one of my comment is now outdated but it should check that the type of the repository is `github` or throw an error